pmlc/dialect/stdx/transforms/i1_to_i32.cc

```diff
@@ -101,10 +112,48 @@ LogicalResult StoreOpI1ToI32::matchAndRewrite(StoreOp storeOp,
   return success();
 }
 
+/// Changes TruncateIOp to i1 to
+/// CmpIOp which is followed by selectOp or
```

Comment:
remove this copy of description, just leave the one above

Reply:
Resolved.
BTW, currently we convert all truncateOp which target to i1 to cmpIOp or their origin integer type. I feel this could only be a current workaround, and not sure if it's a good way to deal with this.

Reply:
Yes, I think it is good for now. I would expect MLIR to fix the i1 issue of spv.SConvert and/or spv.Select.  I will merge this PR but we should think more before we submit to plaidml.